### ISSUE TITLE
AutoField is not using the createAutoField as suggested by uniforms docs

### ISSUE THREAD

**Issue by rafaeltuelho** [aerogear/uniforms-patternfly#74]
Please see this discussion on uniforms github repo: https://github.com/vazco/uniforms/discussions/941#discussioncomment-679631

They suggest defining `AutoField` using `createAutoField` as in https://github.com/vazco/uniforms/blob/7f1fddbf9dea9572835536d321fe4098de60e355/packages/uniforms-unstyled/src/AutoField.tsx#L15

**Comment by wtrocki** [comment]
Yes. We need to refactor our autofield to use helper as other themes.  I'm on PTO next week but will try to see how fast we can do it. Contributions welcome

**Comment by kingsleyzissou** [comment]
Someone mentioned this to me yesterday too. I think we are a bit behind `uniforms`.

I may have some time next week to look at this. If not, contributions are welcome.
